### PR TITLE
refactor(core): delete some unused code

### PR DESCRIPTION
**Pull request by alxhub**
Nothing to see here, just some routine cleanup ;)

### CODE REVIEW

**Comment by JoostK** [comment]
Heya, the View Engine tests appear to be failing!

🙃

**Comment by AndrewKushnir** [comment]
[Global Presubmit](https://test.corp.google.com/ui#id=OCL:409319850:BASE:409320129:1636696160907:d0658fc4).

File: packages/core/src/debug/debug_node.ts [hunk 1]
```diff
@@ -867,18 +717,14 @@ export function getDebugNode__POST_R3__(nativeNode: any): DebugNode|null {
 /**
  * @publicApi
  */
-export const getDebugNode: (nativeNode: any) => DebugNode | null = getDebugNode__PRE_R3__;
-
-
-export function getDebugNodeR2__PRE_R3__(nativeNode: any): DebugNode|null {
-  return getDebugNode__PRE_R3__(nativeNode);
-}
+export const getDebugNode: (nativeNode: any) => DebugNode | null = getDebugNode__POST_R3__;
```
Comment (jessicajaniuk) [inline packages/core/src/debug/debug_node.ts:720]: Maybe add a TODO for this cleanup>

**Review by jessicajaniuk — APPROVED** [review]
LGTM 🍪  🍪  🍪 

Very excited to see this land!

**Review by atscott — APPROVED** [review]

**Comment by IgorMinar** [comment]
@google-cla I signed it!

**Review by AndrewKushnir — APPROVED** [review]

**Comment by alxhub** [comment]
Caretaker: this requires a g3 patch update, as you know :)

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 8c71b9fc42e1aa898200b2b3ebb54d6faf0fa645.

**Pull request closed without merging**